docs/troubleshooting.md

```diff
@@ -34,28 +34,6 @@ Samples that receive less than 5% alignment are skipped for further analysis. Yo
 ### Data organization
 The pipeline can't take a list of multiple input files - it takes a glob expression. If your fastq files are scattered in different paths then we recommend that you generate a directory with symlinked files. If running in paired end mode please make sure that your files are sensibly named so that they can be properly paired. See the previous point.
 
-## UPPMAX specific options
```

Comment:
Why did you remove this section?

Reply:
Because we don't need to do `module load bioinfo-tools` any more, since we are using singularity.

Reply:
Well why do we even have an uppmax config file then? Seem very arbitrary to just remove some documentation but not the actual feature.